back/boxtribute_server/exceptions.py

```diff
@@ -51,6 +51,22 @@ class InvalidTransferAgreement(Exception):
     }
 
 
+class InvalidTransferAgreementState(Exception):
+    extensions = {
+        "code": "BAD_USER_INPUT",
+        "description": "The state of the transfer agreement does not allow the "
+        "requested action.",
```

Comment:
@pylipp can we pass in what "requested action" is? This is not particularly descriptive.

Reply:
I assume this error message would be presented to the FE devs rather than to the end users. The full response from GraphQL looks like
```
{
  "data": {
    "acceptTransferAgreement": null
  },
  "errors": [
    {
      "extensions": {
        "code": "BAD_USER_INPUT",
        "description": "The state of the transfer agreement does not allow the requested action."
      },
      "locations": [
        {
          "column": 3,
          "line": 2
        }
      ],
      "message": "An unknown error occurred.",
      "path": [
        "acceptTransferAgreement"
      ]
    }
  ]
}
```
and here it's visible that `acceptTransferAgreement` was executed.
If we consider using the `description` alone (by displaying it to the end user but really the cause of the error should be fixed during FE development already I think) I can add the action name.